### ISSUE TITLE
[Hygiene] Add a parameter to exclude lists.

### ISSUE THREAD

**Issue by TheImmigrant** [OpenCTI-Platform/connectors#909]
## Use case

Some organizations might need to tight controls on some lists that might not be relevant for them. 
For example, exclude this list.  "Top 1,000,000 most-used sites from Tranco"

## Current Workaround

Manually remove the hygiene label.

## Proposed Solution

Docker parameter to exclude some lists.

## If the feature request is approved, would you be willing to submit a PR?

Yes

**Comment by annoyingapt** [comment]
Yeah the top 1 mill lists are annoying for compromised sites that are being used to host files. Has any progress been made on this? I haven't looked at the code yet, do you have any ideas how to get this to work?
